fuzzers/binary_only/fuzzbench_fork_qemu/src/fuzzer.rs

```diff
@@ -335,7 +333,7 @@ fn fuzz(
         }
 
         unsafe {
-            qemu.write_mem_unchecked(input_addr, buf);
+            qemu.write_mem(input_addr, buf).expect("qemu write failed.");
```

Comment:
We may want to leave this unchecked for performance reasons, cc @rmalmain

Reply:
using checked write out of the fuzzing loop is fine.
in fuzzing loop we should avoid the checked version.

Reply:
It's in the fuzzing loop, it's the harness right?

Reply:
should i leave this unchecked then?

Reply:
Yes and if you want, maybe leave a short comment like:
```
// # Safety
// The input buffer size is checked above. We use `mem_unchecked` for performance reasons.
```

Reply:
Thanks!